roles/pgbench/tasks/run_workload.yml

```diff
@@ -5,6 +5,46 @@
 - name: Init client ready state to empty
   command: "redis-cli del pgb_client_ready"
 
+- name: Calculate timeouts
+  include_tasks: calculate_timeouts.yml
+
+- name: Pre-pull pgbench container image
```

Comment:
Do we need this? This feels like an additonal step.

Reply:
Image download time is harder to predict. This is done as a pre-pull so that it can have its own timeout value as an operation separate from starting the pod(s).

Reply:
Look at it this way -- If I _don't_ separate out the pre-pull, then I need to add the tolerated maximum amount of time for the image pull (currently about 11 minutes x number of databases) to the load_timeout. This permanently expands the load_timeout by this added image pull tolerance time, regardless of whether the image is already present on the k8s nodes. With the submitted design, if the images _are_ already present on the system, then the pre-pull jobs will complete very quickly and we will move on to the pgbench test job(s) task, which then has calculated timeouts relevant to only how long it should take to start/load the pods given that the image is already local to the nodes.